README.md

```diff
@@ -166,11 +166,17 @@ Name | Description
 stream    | Stream id to subscribe to
 apiKey   | User key or stream key that authorizes the subscription. If defined, overrides the client's `apiKey`.
 partition | Partition number to subscribe to. Defaults to the default partition (0).
-resend_all | Set to `true` if you want all the messages for the stream resent from the earliest available message.
-resend_last | Resend the previous `N` messages.
-resend_from | Resend from a specific message number.
-resend_from_time | Resend from a specific Date (or millisecond timestamp).
-resend_to | Can be used in conjunction with `resend_from` to limit the end of the resend. By default it is the newest message.
+resend | Object defining the resend options that are specified below.
+
+#### Resend options
+
+Name | Description
+---- | -----------
+last | Resend the previous `N` messages.
+from | Resend from a specific message reference. This option must be an object that defines a `timestamp` and can optionally define a `sequenceNumber`.
```

Comment:
Instead of this table, examples of the 3 main cases (last, from, from-to) might be more readable, due to the various interdependencies?

```
    // Resend N most recent messages
    resend: {
        last: 10,
    }

    // Resend from a specific message reference up to the newest message
    resend: {
        from: {
            timestamp: 12345,
            sequenceNumber: 0, // optional
        }
        publisher: 'publisherId', // optional
        msgChainId: 'msgChainId', // optional
    }

    // Resend a limited range of messages
    resend: {
        from: {
            timestamp: 12345,
            sequenceNumber: 0, // optional
        },
        to: {
            timestamp: 54321,
            sequenceNumber: 0, // optional
        },
        publisher: 'publisherId', // optional
        msgChainId: 'msgChainId', // optional
    }
```